docker/zap-baseline.py

```diff
@@ -425,7 +430,7 @@ def main(argv):
 
     if fail_count > 0:
         sys.exit(1)
-    elif warn_count > 0:
+    elif (not ignore_warn) and warn_count > 0:        
```

Comment:
The trailing white space seems unnecessary.

Reply:
yeah oops - fixed in subsequent commit. I like to press space when I am thinking...